tap_facebook/__init__.py

```diff
@@ -252,13 +291,8 @@ def job_params(self):
             }
             buffered_start_date = buffered_start_date.add(days=1)
 
-    @backoff.on_exception(
-        backoff.constant,
-        InsightsJobTimeout,
-        max_tries=2,
-        interval=0,
-        on_backoff=log_insights_retry_attempt
-    )
+    @retry_pattern(backoff.constant, InsightsJobTimeout, max_tries=2)
+    @retry_pattern(backoff.expo, FacebookRequestError, max_tries=5, factor=5)
```

Comment:
What does having two retry patterns do?

Reply:
The original pattern was using a single, instant retry since the timeout usually resolves on the second attempt. Though it wouldn't be too crazy for them to both use Exponential. I can consolidate them.